docs/reference/concepts/05-events.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 1
+id: events
+---
+
+# Events
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Events allow enable the ability to react to state changes in the provider or underlying flag management system.
```

Comment:
```suggestion
Events allow  _application authors_ to add logic that reacts to state changes exposed by the provider or underlying flag management system.
```

Reply:
I think events are helpful not to just application authors, so I didn't mention a particular persona.

Reply:
Sure, that's fine but the `Events allow enable` section needs to be rephrased.

Reply:
Oh, right. Thanks.